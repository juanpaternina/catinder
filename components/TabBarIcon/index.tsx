import { View } from "react-native";

export default function TabBarIcon(props: { children: React.ReactNode }) {
  return (
    <View
      testID="tab-bar-icon"
      style={{
        marginBottom: -13,
        height: 64,
        justifyContent: "center",
      }}
    >
      {props.children}
    </View>
  );
}
