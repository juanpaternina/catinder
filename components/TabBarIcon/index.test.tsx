import { render, screen } from "@testing-library/react-native";
import TabBarIcon from "./index";

describe("TabBarIcon", () => {
  it("renders correctly", () => {
    render(<TabBarIcon>Test</TabBarIcon>);
    expect(screen.getByTestId("tab-bar-icon")).toBeTruthy();
    expect(screen).toMatchSnapshot();
  });
});
