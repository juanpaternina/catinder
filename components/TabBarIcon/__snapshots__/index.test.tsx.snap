// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabBarIcon renders correctly 1`] = `
<View
  style={
    {
      "height": 64,
      "justifyContent": "center",
      "marginBottom": -13,
    }
  }
  testID="tab-bar-icon"
>
  Test
</View>
`;
